media-libs/portsmf/portsmf-239.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="The Tenacity fork of PortSMF, a Standard MIDI File library"
+HOMEPAGE="https://github.com/tenacityteam/portsmf"
+SRC_URI="
+	https://github.com/tenacityteam/portsmf/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+"
+LICENSE="MIT"
+SLOT="0/1"  # SOVERSION in CMakeLists.txt / SONAME suffix
```

Comment:
I wonder if you need to specify SONAME at all here? Are you aware of upstream breaking ABI with releases?

Reply:
I'm not aware of them tracking ABI breaks if they do, but theres an SONAME, so I figured it's better safe than sorry

Reply:
In general, I think it's worth specifying it going forward for new ebuilds, yes. Because then we avoid the chicken-and-egg issue where when abi does break, some poor soul (usually me!) has to go around adding :=s